Lpd 55155

georgel-pop-lr opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

andresfulla commented:
Reviewing! <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

sandrodw3 commented:
Pull request submitted to https://github.com/p2kmgcl/liferay-portal/pull/497. [See changes here.](https://github.com/sandrodw3/liferay-portal/compare/e62e1e24806b~2...e62e1e24806b) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

lfbesada committed:
LPD-54517 Move the code to create the display page template when getting the URL if it does not exist, so it applies when editing the entries too, no logic changes

lfbesada committed:
LPD-54517 Adapt the edit usage

lfbesada committed:
LPD-54517 Not needed

# Conflicts:
#	modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/BaseSectionDisplayContext.java

lfbesada committed:
LPD-54517 Add a struts action to edit the structure's display page, creating it if does not exist

lfbesada committed:
LPD-54517 Add an integration test to assert EditStructureDisplayPageStrutsAction

lfbesada committed:
LPD-54517 Add a company to prevent the _systemGroupsMap cache

lfbesada committed:
LPD-54517 Add a type setting to the draft layout to mark the autogenerated layouts

lfbesada committed:
LPD-54517 Remove the type settings when a draft layout set its status to draft as it means it is being customized

lfbesada committed:
LPD-54517 Baseline

lfbesada committed:
LPD-54517 Fix typo, we must update the draft layout type settings

lfbesada committed:
LPD-54517 Add asserts to ensure the draft layout autogenerated type setting

lfbesada committed:
LPD-54517 Add required dependency

lfbesada committed:
LPD-54517 Add a strut action to reset de structure's experience in case of not customized

lfbesada committed:
LPD-54517 Add an integration test to assert ResetStructureDisplayPageStrutsAction

lfbesada committed:
LPD-54517 Add the struts action URLs to the context

georgel-pop-lr committed:
LPD-54033 Add the new CardStyleModal style design to MarketplacePresentationModal.tsx https://github.com/liferay/clay/pull/5992

georgel-pop-lr committed:
LPD-54033 Add the Customize Experience button

pietrobocale committed:
LPD-53243 Update custom-suite to add the possibility to decide if want integration

georgel-pop-lr committed:
LPD-54033 Add and use NewFeatureModal.tsx

georgel-pop-lr committed:
LPD-54033 Add new test for NewFeatureModal.test.js

georgel-pop-lr committed:
LPD-54033 Add the customize-experience key

georgel-pop-lr committed:
LPD-54033 buildLang

victorg1991 committed:
LPD-55152 This should be a button

victorg1991 committed:
LPD-55152 Simplify

victorg1991 committed:
LPD-55152 Rename to a more generic name

victorg1991 committed:
LPD-55152 Include urls in the config

victorg1991 committed:
LPD-55152 Navigate to the content editor

victorg1991 committed:
LPD-55152 Redirect to content editor or show modal depending on the conditions

victorg1991 committed:
LPD-55152 Language keys

victorg1991 committed:
LPD-55152 buildLang

victorg1991 committed:
LPD-55152 Reset display page when structure changes

victorg1991 committed:
LPD-55152 Register in the state when there are changed that are not saved

victorg1991 committed:
LPD-55152 Ask for publish the structure when there are changes

victorg1991 committed:
LPD-55152 Use p_l_back_url

joshitagaki committed:
LPD-53243 Rename to integration.suite.enabled

victorg1991 committed:
LPD-55152 Add test

victorg1991 committed:
LPD-55152 Extract so it can be reused

victorg1991 committed:
LPD-55152 Fix typescript

victorg1991 committed:
LPD-55152 Fix test

joshitagaki committed:
LPD-53243 Fix formatting

tinatian committed:
LPD-55293 Do not fail fast on tck test task to make sure all the logic is executed

ling-alan-huang committed:
LPD-43765 SF, use TransformUtil.transform

ling-alan-huang committed:
LPD-43765 Add dependencies

ling-alan-huang committed:
LPD-43765 SF, use TransformUtil.transform

ling-alan-huang committed:
LPD-43765 SF, use TransformUtil.transform

ling-alan-huang committed:
LPD-43765 Add dependencies

ling-alan-huang committed:
LPD-43765 SF, use TransformUtil.transform

ling-alan-huang committed:
LPD-43765 SF, use TransformUtil.transform

ling-alan-huang committed:
LPD-43765 SF, use TransformUtil.transform

victorg1991 committed:
LPD-55152 Fix test

ling-alan-huang committed:
LPD-43765 SF, use TransformUtil.transform

georgel-pop-lr committed:
LPD-55155 Remove space-list fragment from site-initializer-cms

georgel-pop-lr committed:
LPD-55155 Add space list to site.cms.site.initializer and render the space name

georgel-pop-lr committed:
LPD-54033 Rename to CardStyleModal.test.js

georgel-pop-lr committed:
LPD-55155 Adapt test

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.
